<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEmployeeDialog</class>
 <widget class="QDialog" name="addEmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Employee Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QRadioButton" name="hourlyRadioButton">
        <property name="text">
         <string>Hourly</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="monthlyRadioButton">
        <property name="text">
         <string>Monthly</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="salesManRadioButton">
        <property name="text">
         <string>Salesman</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="ssnLabel">
            <property name="text">
             <string>SSN :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="monthlyCompLabel">
            <property name="text">
             <string>Monthly Comp. :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hourlyCompLabel">
            <property name="text">
             <string>Hourly Comp. :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="doneHoursLabel">
            <property name="text">
             <string>Done Hours:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bonusLabel">
            <property name="text">
             <string>Bonus :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="realizedOutcomeLabel">
            <property name="text">
             <string>Realized Outcome :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="ssnLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="monthlyCompLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="hourlyCompLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="doneHoursLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="bonusLineEdit"/>
          </item>
          <item>
           <widget class="QLineEdit" name="realizedOutcomeLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okPushButton">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelPushButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
